src/plugins/eqht-css/reports/1-assessment-wcag21.json

```
@@ -0,0 +1,629 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:hasPart": [
+      "_:non-text-content_itm1"
+    ],
+    "dct:references": "https://wet-boew.github.io/wet-boew/demos/eqht-css/eqht-css-en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Only on the component 'Equal height css', not its container neither any other sibling component.",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": ".wb-eght-css"
+    }
+  },
+
+  "earl:assertedBy": {
+    "foaf:name": "Service Canada - Principal Publisher",
+    "foaf:homepage": "https://github.com/ServiceCanada",
+    "@type": ["earl:Assertor", "foaf:Organization"],
+    "earl:mainAssertor": {
+      "foaf:name": "Hongbin yu (Github: @hongbinyu413)",
+      "foaf:homepage": "https://github.com/hongbinyu413",
+      "@type": ["earl:Assertor", "foaf:Person"]
+    }
+  },
+
+  "dct:date": "2022-07-06",
+
+  "dct:description": "Partial evaluation focused on the 'equal height' component. Evaluation of all Success Criteria at Level AA of WCAG 2.1",
+
+  "dct:source": "act:rulesets/wcag2x/wcag21_all_levelAA.json",
+  "acr:standard": "https://www.w3.org/TR/WCAG21",
+  "acr:conformanceOption": "act:standard/profiles/wcag#levelAA",
+
+  "earl:result": [
+    {
+      "earl:test": "WCAG21:non-text-content",
+      "earl:outcome": "earl:passed",
+      "earl:subject": {
+        "@id": "_:non-text-content_itm1",
+        "dct:isPartOf": "_:subject",
+        "@type": [ "earl:TestSubject", "schema:WebPageElement" ],
+        "dct:description": "Expand/collapse arrow"
+      },
```

This need to be updated to match the tested page.

```suggestion
      "earl:subject": "_:subject",
```